source/Public/SQL Database/Get-FabricSQLDatabase.ps1

```diff
@@ -25,33 +25,37 @@ function Get-FabricSQLDatabase {
     The value for SQLDatabaseID is a GUID. An example of a GUID is '12345678-1234-1234-1234-123456789012'.
 
 .EXAMPLE
-    Get-FabricSQLDatabase `
-        -WorkspaceId '12345678-1234-1234-1234-123456789012' `
-        -SQLDatabaseName 'MySQLDatabase'
+    $FabricSQLDatabaseConfig = @{
+        WorkspaceId = '12345678-1234-1234-1234-123456789012'
+        SQLDatabaseName = 'MySQLDatabase'
+    }
+    Get-FabricSQLDatabase @FabricSQLDatabaseConfig
 
-    This example will retrieve the SQLDatabase with the name 'MySQLDatabase'.
+    Returns the details of the Fabric SQL Database with the name 'MySQLDatabase' in the workspace that is specified by the WorkspaceId.
 
 .EXAMPLE
     Get-FabricSQLDatabase -WorkspaceId '12345678-1234-1234-1234-123456789012'
 
-    This example will retrieve all SQLDatabases in the workspace that is specified
-    by the WorkspaceId.
+    Returns the details of the Fabric SQL Databases in the workspace that is specified by the WorkspaceId.
 
 .EXAMPLE
-    Get-FabricSQLDatabase `
-        -WorkspaceId '12345678-1234-1234-1234-123456789012' `
-        -SQLDatabaseId '12345678-1234-1234-1234-123456789012'
+    $FabricSQLDatabaseConfig = @{
+        WorkspaceId = '12345678-1234-1234-1234-123456789012'
+        -SQLDatabaseId = '12345678-1234-1234-1234-123456789012'
+    }
+    Get-FabricSQLDatabase @FabricSQLDatabaseConfig
 
-    This example will retrieve the SQLDatabase with the ID '12345678-1234-1234-1234-123456789012'.
+    Returns the details of the Fabric SQL Database with the ID '12345678-1234-1234-1234-123456789012' from the workspace with the ID '12345678-1234-1234-1234-123456789012'.
 
 .EXAMPLE
-    Get-FabricWorkspace -WorkspaceName 'MsLearn-dev' | Get-FabricSQLDatabase
+    Get-FabricWorkspace -WorkspaceName prod-workspace | Get-FabricSQLDatabase
 
-    This example will retrieve all SQLDatabases from the workspace with the name 'MsLearn-dev' (using the pipeline).
+    Returns the details of the Fabric SQL Databases in the prod-workspace workspace.
 
 .NOTES
     Revision History:
         - 2025-03-06 - KNO: Init version of the function
+        - 2025-06-14 - Update the exampes to remove backticks
```

Comment:
```suggestion
        - 2025-06-14 - Update the examples to remove backticks
```

Reply:
nice spot, thank you. PRs are brilliant for checking these sort of things.